GBL/include/GblTrackSegmentController.h

```diff
@@ -47,7 +47,7 @@ namespace genfit {
     
     virtual GblTrackSegmentController* clone() const {return new GblTrackSegmentController(*this);}
     
-    virtual void controlTrackSegment(TVector3, TVector3, GblFitter *) {;}
```

Comment:
I know that the comments are missing in genfit all over the place. But maybe (as you have touched this code), could you at least add some variable names? You can e.g. comment them out to not get unused arguments.
Another comment: if the function is empty (and is probably overridden anyways): do you want to make it fully virtual (`= 0`). So you can use the variable names again.

Reply:
Good points. Fixed.